internal/gapicgen/generator/genproto.go

```diff
@@ -110,7 +123,7 @@ func regenGenproto(ctx context.Context, genprotoDir, googleapisDir, protoDir str
 	// Run protoc on all protos of all packages.
 	grp, _ := errgroup.WithContext(ctx)
 	for pkg, fnames := range pkgFiles {
-		if !strings.HasPrefix(pkg, "google.golang.org/genproto") || denylist[pkg] {
+		if !strings.HasPrefix(pkg, "google.golang.org/genproto") || denylist[pkg] || hasPrefix(pkg, skipPrefixes) {
```

Comment:
If `skipPrefixes` gets larger, it would be more efficient to preemptively remove strings that match the prefix from `pkgFiles` rather than checking it everytime in this for loop. Considering the size of `skipPrefixes` for now, I don't think it's a blocking optimization.

Reply:
I agree on both counts. Going for simplicity for now.